Test/Units/CommandTestCase.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace atoum\AtoumBundle\Test\Units;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CommandTestCase extends Test
+{
+    /**
+     * Create a CommandTester instance to test commands
+     *
+     * @param ContainerAwareCommand $command
+     *
+     * @return CommandTester
+     */
+    public function createCommandTester(ContainerAwareCommand $command)
```

Comment:
I don't think using `ContainerAwareCommand` is required as you never use any of its method.

Perhaps just type hinting on commands base class is enough ?